app/views/favorites/index.html.erb

```diff
@@ -0,0 +1,7 @@
+
+<% @fav_pets.each do |pet| %>
+  <section id="favorite<%= pet.id %>">
+    <%= image_tag(pet.image, {:style => "width:200px;"}) %>
```

Comment:
O.pportunity to refactor: image style is showing up weird in server. Look it over when you get a chance.

Reply:
Will do.